composer.json

```diff
@@ -3,37 +3,35 @@
     "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
     "type": "project",
     "require": {
-        "php": ">=5.6.0",
+        "php": ">=7.1.3",
```

Comment:
```suggestion
        "php": ">=7.1.0",
```

Reply:
not sure it's worth as most of the libraries set 7.1.3 to be compatible with Symfony 4